embeddings/pipeline/preprocessing_pipeline.py

```diff
@@ -111,21 +127,27 @@ def __init__(
         input_column_name: str,
         target_column_name: str,
         datasets_path: Path = DATASET_PATH,
-        sample_missing_splits: bool = True,
+        sample_missing_splits: Tuple[float, float] = (0.1, 0.1),
```

Comment:
Same as previously: In my view sampling in pipelines should be optional instead of forcing sampling.